client/dashboard.html

```diff
@@ -179,6 +181,26 @@
 
    }
 
+   function likario(postid){
+    var xhttp = new XMLHttpRequest();
+    xhttp.onreadystatechange = function() {
+       if (this.readyState == 4 && this.status == 200) {
+         var result = this.response;
+         console.log(result);
+         if(result == "true")
+         {

```

Comment:
Open statement blocks on the same line as their declaration (Code Style Guide 4)